tests/flytekit/unit/tools/test_script_mode.py

```diff
@@ -237,3 +240,70 @@ def test_get_all_modules(tmp_path):
     # Workflow exists, so it is imported
     workflow_file.write_text(WORKFLOW_CONTENT)
     assert n_sys_modules + 1 == len(get_all_modules(os.fspath(source_dir), "my_workflows.main"))
+
+
+def test_list_imported_modules_as_files():
+
+    source_path = "/home/username/project"
```

Comment:
For windows CI to pass, I think we'll either need to use `os.path.join` or `pathlib.Path` to define the paths.

(Windows uses `\` instead of `/`)